.travis.yml

```diff
@@ -42,7 +44,6 @@ env:
 matrix:
   allow_failures:
     - env: ROS_DISTRO=indigo PRERELEASE=true PRERELEASE_REPONAME=industrial_core PRERELEASE_DOWNSTREAM_DEPTH=0
-    - env: ROS_DISTRO=indigo PRERELEASE=true PRERELEASE_REPONAME=ros_canopen
```

Comment:
Aside from the recent discussion development in #64, the commit that gets used in the ros prerelease test is the latest in the development branch listed at `rosdistro`, which means that ros prerelease test as of now does not test your pull request by default. So until #64 is resolved we better not use prereleae for checking regression, no?